src/StackAdmin/Azs.Compute.Admin/Tests/packages.config

```diff
@@ -1,5 +1,5 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
   <package id="Microsoft.AzureStack.Management.Compute.Admin" version="0.3.0-preview" targetFramework="net45" />
-  <package id="Microsoft.Rest.ClientRuntime.Azure.TestFramework" version="1.6.0" targetFramework="net45" />
```

Comment:
This was removed by Yanni, so technically don't need.  I wish Azure-PowerShell would move to the new version as it gives more information if there is a problem with the recordings.